superlists/functional_tests.py

```diff
@@ -18,15 +18,20 @@ def test_can_start_a_list_and_retrieve_it_later(self):
 
 		# I notice the page title and header mention to-do lists
 		self.assertIn('To-Do', self.browser.title)
-		
+		header_text= self.browser.find_element_by_tag_name('h1').text
+		self.assertEqual('To-Do lists', header_text)
 
 		# I am invited to enter a to-do item straight away
-		
+		inputbox=self.browser.find_element_by_id('id_new_item')
```

Comment:
**Note1:** Readability issue space before and after `=` sign.
`inputbox=self.browser.find_element_by_id('id_new_item')` --> `inputbox = self.browser.find_element_by_id('id_new_item')`